package.json

```diff
@@ -19,6 +19,7 @@
 	"dependencies": {
 		"glob": "^7.1.6",
 		"knockout": "^3.5.1",
+		"meriyah": "^3.1.6",
```

Comment:
Didn't we have a dependency on acorn to remove?

Reply:
No, we did actually not have acorn as a dependency, oddly enough